lib/experimental/interactivity-api/modules.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Interactive modules.
+ *
+ * @package Gutenberg
+ * @subpackage Interactivity API
+ */
+
+/**
+ * Register the `@wordpress/interactivity` module.
+ */
+function gutenberg_register_interactivity_module() {
+	gutenberg_register_module(
```

Comment:
We should have a way in our code to have an array of "esm modules" and have these be registered automatically (similarly to the other types of packages we have: non modules)